tests/ext-popup.test.js

```diff
@@ -188,20 +188,11 @@ test('Error is shown in popup UI while wrong parameter is passed to handleEvent'
 });
 
 test('clicking on "view activity logs page" button should open the activitylog page', async () => {
-  document.body.innerHTML = popupBody;
```

Comment:
Could it be that this is not optional? Overwriting the document with the initial HTML would clear any DOM changes and event listeners from previous tests, if any.

Reply:
It's possible, but the test result does not change regardless of this call, however I change the rest of the test.